test/mock/MockBlastERC20.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import {YieldMode} from "../../../contracts/interfaces/IERC20Rebasing.sol";
+
+contract MockERC20 is ERC20 {
```

Comment:
the name of the file is MockBlastERC20 so perhaps this should be MockBlastERC20?

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/ce33daf9cf033280516e78d9ac601a8b5da5b6fb